presto-main/src/main/java/com/facebook/presto/operator/PipelineContext.java

```diff
@@ -417,9 +425,14 @@ public PipelineStats getPipelineStats()
                 .filter(driver -> driver.getEndTime() == null && driver.getStartTime() != null)
                 .flatMap(driver -> driver.getBlockedReasons().stream())
                 .collect(toImmutableSet());
-        boolean fullyBlocked = drivers.stream()
+
+        Set<DriverStats> runningDriverStats = drivers.stream()
                 .filter(driver -> driver.getEndTime() == null && driver.getStartTime() != null)
+                .filter(DriverStats::isFullyBlocked)
```

Comment:
This seems to be inverted. The running list should only include "not fully blocked".

Is this covered by any tests?